packages/sdk-js/src/GrowthBook.ts

```diff
@@ -1146,8 +1195,34 @@ export class GrowthBook<
     else console.log(msg, ctx);
   }
 
+  public getDeferredTrackingCalls() {
+    return btoa(JSON.stringify(this._deferredTrackingCalls));
+  }
+
+  public fireDeferredTrackingCalls(data: string) {
+    const calls = JSON.parse(atob(data));
+    if (!Array.isArray(calls)) throw new Error("Invalid tracking data");
+    calls.forEach((call: TrackingData) => {
+      if (!call || !call.experiment || !call.result) {
+        throw new Error("Invalid tracking data");
+      }
+      this._track(call.experiment, call.result);
+    });
```

Comment:
Could create a flush function for `deferredTrackingCalls` and call it here. Could also be used if we decided to leverage deferredTrackingCalls for the base case (inBrowser)